requirements.txt

```diff
@@ -1 +1,2 @@
 simpleeval>=0.9.10
+wcmatch >= 8.1.2
```

Comment:
@Zxilly plz remove `fnmatch` from dependency.

Reply:
`fnmatch` is a python builtin library.